### PR TITLE
Add specs for Wishlist model validations

### LINKED ISSUE #65: Add uniqueness and presence validations for wishlists' token

### PR DESCRIPTION
**Pull request by student-tomasz**
Closes #65

### CODE REVIEW

**Comment by shadre** [comment]
:+1:

File: spec/models/wishlist_spec.rb [hunk 1]
```diff
@@ -4,7 +4,26 @@
...
+    it "has an optional description" do
+      expect(build(:wishlist, description: nil)).to be_valid
+    end
+
+    it "sets token on its own if none provided" do
+      expect(build(:wishlist)).not_to be_nil
+    end
+
+    it "accepts a token passed in initialization" do
+      expect(build(:wishlist, token: 'unique-token')).to be_valid
```
Comment (Kacper3331) [inline spec/models/wishlist_spec.rb]: It's just my suggestion: change single quotation marks to double 😄

Comment (student-tomasz) [inline spec/models/wishlist_spec.rb]: I didn't notice RuboCop stopped reporting warnings, sorry for that.

File: app/models/concerns/tokenable.rb [hunk 2]
```diff
@@ -2,12 +2,13 @@ module Tokenable
...
-    before_create :set_token
+    after_initialize :set_token
   end
 
   protected
 
   def set_token
+    return if self.token
     self.token = loop do
       random_token = SecureRandom.uuid
```
Comment (gotar) [inline app/models/concerns/tokenable.rb:13]: uuid if unique by default (it almost impossible to get the same one) no reason to check it in loop

**Comment by gotar** [comment]
nice, good enough to go live

**Pull request merged**